src/clients/node/package.json

```diff
@@ -34,12 +25,10 @@
   "scripts": {
     "benchmark": "./scripts/benchmark.sh",
     "test": "node dist/test",
-    "postinstall": "npm run download_node_headers && npm run build_lib",
```

Comment:
Fed's idea: delete the unneeded binaries here

Reply:
I'm going to leave this open, so we can discuss some pros / cons. The current approach is:
* We ship binaries for each support platform
* At runtime, (see index.ts), we determine which one to load. There's a hack in here that still needs to be dealt with re detecting musl vs glibc.

The alternative:
* On postinstall, do the detection, and remove the other binaries, so there's only eg `dist/bin/client.node` and nothing is required at runtime.
  * The downside with this, is it would mean you can't `cp` your project between different machines
  * Actually thinking about it, it would also mean trouble volume mounting and working with the same folder in Docker if you're going between musl / libc / mac.
  * It would also require a separate script. This can be in Node though, so at least we avoid cross-platform sh pitfalls
* Saves a few MB from not having them around, and no runtime code required...

Cc @matklad

Reply:
A third alternative (and that's how I believe node projects usually work) is to _download_ correct binary in postinstall from some well-known location (eg, github artifcat). 

I'd say lets keep it simple and just ship everything for now.

Reply:
> Actually thinking about it, it would also mean trouble volume mounting and working with the same folder in Docker if you're going between musl / libc / mac.

If there is an issue that wouldn't work for us in this scenario it would be an issue that wouldn't work for most software either. Most systems don't include all variations of their binaries for all architectures.

Reply:
Yeah the download-in-postinstall is pretty common but agreed about keeping it simple. It really boils down to if we think checking the arch, platform and libc on each `require` of the package is acceptable or not. I do think it's the simplest.